woocommerce/woocommerce
# fix: jest configuration changes to allow is-plain-obj esm

### All Submissions:






### Changes proposed in this Pull Request:

Due to some [upstream changes](https://github.com/WordPress/gutenberg/pull/42508) including a package that uses ESM, jest needs to transform such files before it can be run. 

### How to test the changes in this Pull Request:
Purely a dev env change; Jest tests should pass both in CI and locally

1. Try commands such as `git clean -fdx && pnpm i && pnpm run build && cd ./plugins/woocommerce-admin && pnpm run test`
2. And also attempt to run tests within the packages themselves


### Other information:




### FOR PR REVIEWER ONLY:
